tools/filters/gff/gtf_filter_by_attribute_values_list.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="gtf_filter_by_attribute_values_list" name="Filter GTF data by attribute values_list" version="0.1">
     <description></description>
     <command interpreter="python">
```

Comment:
Should we remove `interpreter`? 

Are there any rules regarding the profile of "internal" tools? Is the default also 16.01 or is it the current release?

Reply:
Yes, please remove the interpreter tag. Internal tools aren't different from other tools, if they don't specify a profile they will use the old defaults.